index.html

```diff
@@ -64,6 +64,32 @@ <h1 class="cTitle">300+</h1>
       </section>
 
       <section><!--sUCCESS STORY --></section>
+    <section class="suc-sto">
```

Comment:
Can you please delete section with comment? Then i will merge it.

Reply:
@firity did you format your code?

Reply:
Thank you for review.I didn't deleted but edit it,